Add client certificate rest check example

0lvin opened the pull request:
Author: Denis Pauk (denisp@cloudify.co)

Code is based on https://github.com/sevcsik/client-certificate-demo client certification validation example.

EarthmanT commented:
1. Is nodejs directory from another git? Why does it need its own gitignore? Can you remove it?
2. Do we need package lock .json?
3. Can you take the cert data out of the blueprint and maybe add it as an input file ?

Thanks!

0lvin commented:
> Is nodejs directory from another git? 

I have remove not required files, so we have only server.js and  package.json as related to nodejs. And looks like it will be never updated separately from blueprints.

> Why does it need its own gitignore? Can you remove it? Do we need package lock .json?

I have removed lock file and gitignore.

> Can you take the cert data out of the blueprint and maybe add it as an input file ?

I have moved raw certificates to separate input file.
